index.html

```diff
@@ -47,23 +56,51 @@
         position: sticky;
         top: 0;
       }
+      .personal-info {
+        margin-left: 25px;
+      }
+      @media only screen and (max-width: 768px) {
+          /* For mobile phones: */
+          [class*="column-"] {
+            width: 100%;
+          }
+          .small-screen--hide {
+            display: none;
+          }
+          .small-screen--one-whole {
+            width: 100%;
+          }
+          .small-screen--container {
```

Comment:
I think we have a consistency problem with the name

For one property I see you are using after `--` a noun
e.g. : **.small-screen--profile-card**

but you are using after `--`  a modifier(adjective) too.
e.g.: **.small-screen--hide**


I suggest to use a naming system like [BEM](http://getbem.com/naming/), you could also create your own one but need to be consistent. If need any help, let me know